utils/generate-groups.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
```

Comment:
Looks like this script was copied from https://github.com/kubernetes/code-generator/blob/master/generate-groups.sh.

If so, please modify the LICENSE file and claim the copyright.

Reply:
See the bottom of https://github.com/apache/apisix-dashboard/blob/master/LICENSE for more details.

Reply:
:joy: it was kubernetes license, but it can't pass CI license-check

Reply:
It's also Apache License but coprighted to Kubernetes Authors. @dmsolr Could you help to help us to adjust this?